.travis.yml

```diff
@@ -17,14 +17,14 @@ services:
   - cassandra
 
 before_script:
+  - npm install -g npm@latest
```

Comment:
Am I correct in assuming that this will not update npm before after the normal `npm install` have run? If so that's 👍